src/en/troubleshooting-logs-audit.md

```diff
@@ -0,0 +1,28 @@
+Title: Audit logging
+
+# Audit logging
+
+Juju audit logging provides a chronological account of all events by capturing
+invoked user commands. These logs reside on the controller involved in the
+transmission of commands affecting the Juju client, Juju machines, and the
+controller itself.
+
+See [Juju log][logs] documentation to learn about standard Juju logging.
```

Comment:
I wouldn't put this here. In fact, it would be best to omit all together and include all this stuff on the logging page

Reply:
I didn't do that because the regular log page is long. I also already refer readers to a separate page for remote logging.

Reply:
Done. I also moved remote logging.